src/interfaces/card.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface ICardProps {
```

Comment:
This Card component is pretty simple so we can easily add support for html props:
`export interface ICardProps  extends React.HTMLAttributes<HTMLDivElement> {}`

Also, please add:
```
export var Card: React.ComponentType<ICardProps>;
```

Reply:
done at e57e61ea1e6ba66aa5c16156a890aba32e1c1e42